Chocolate Classification and Prediction/Model/README.md

```diff
@@ -0,0 +1,39 @@
+### Chocolate Classifiaction and Prediction
+
+## GOAL-
+
+Classify the varieties of the chocolates and predict them accordingly.
+
+## DATASET-
+
+[https://www.kaggle.com/datasets/siddharthmandgi/chocolate-classification](https://www.kaggle.com/datasets/siddharthmandgi/chocolate-classification)
+
+## DESCRIPTION-
+
+Main idea is to train a model that will predict ,which type /class of Chocolate is the onw which is asked to machine in form of a new data feed. Model in this project deal with the binary classified training dataset of the dark and white chocolate.
+
+## WORK DONE
+
+- Analysed and preprocessed the data by eliminating the files/data which is irrelevant o feed in the trainf/validation set.
+- Did Data augmentation and to increase the training and validation set data.
+- Trained a Convolutional Neural Network and fitted the dataset into it.
+
+## LIBRARIES/FRAMEWORK NEEDED
+
+- NumPy
+- Matplotlib
+- PIL
+- TensorFlow
+
+## PLOTS
+
+!(https://drive.google.com/file/d/1xY8Fw7ifkY-N1tGa6axgQ0eJxaVSM0Ix/view?usp=sharing)

```

Comment:
You can directly put the visualization images in the README.md file.
For example, `[](Image url)` You can fetch the URL from the Images folder where you have uploaded the images for this project.

Reply:
i tried doing it in the latest attempts...but it not image is not showing directly in the README.md file.

Reply:
Try to use this tag,
`<img src = "image url">`

Reply:
Or you can use this also,
`![](image url)`

My bad, I missed the `!` sign.